server/odc-service/src/main/java/com/oceanbase/odc/service/common/util/AlarmHelper.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.common.util;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import com.oceanbase.odc.core.alarm.AlarmUtils;
+import com.oceanbase.odc.core.alarm.AlarmUtils.AlarmMessageBuilder;
+import com.oceanbase.odc.core.shared.Verify;
+import com.oceanbase.odc.metadb.schedule.ScheduleEntity;
+import com.oceanbase.odc.service.connection.ConnectionService;
+import com.oceanbase.odc.service.connection.model.ConnectionConfig;
+import com.oceanbase.odc.service.schedule.ScheduleService;
+import com.oceanbase.odc.service.schedule.ScheduleTaskService;
+import com.oceanbase.odc.service.schedule.model.ScheduleTask;
+
+import lombok.NonNull;
+
+public final class AlarmHelper {
```

Comment:
it seems an utils only for task framework, not for common scenario, may move to task module.

Reply:
> it seems an utils only for task framework, not for common scenario, may move to task module.
at present, only the content of the task framework is resolved, but subsequent alarms will expand to other types of content